doc/life_of_a_language_feature.md

```diff
@@ -1,116 +1,141 @@
+Markdown format](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown) format, and must use the `.md` extension.
+  
+Proposals may be iterated on in-place.
 
 For smaller and non-controversial features, we will sometimes skip this step and
-proceed directly to the [Acceptance and
-implementation](#acceptance-and-implementation) phase.
+proceed directly to the [Acceptance](#acceptance) phase.
 
-### External (outside of the language team) feedback
+### External (outside of the language team) Feedback
 
-We expect to use the github issue tracker as the primary place for accepting
-feedback on proposals, solicited or unsolicited. If we solicit feedback, we
-anticipate opening an issue for discussion and feedback, but also encouraging
-filing and splitting off different issues for different threads of discussion.
+We use the Github issue tracker as the primary place for accepting feedback on
+proposals, solicited or unsolicited. 
 
-We generally expect to formally solicit at least one round of feedback on
-significant changes.
+If we solicit feedback, we will open an issue for discussion and feedback. We
+highly recommend filing and splitting off different issues for different threads
+of discussion. We generally expect to formally solicit at least one round of
+feedback on significant changes.
 
-## Acceptance and implementation
+## Acceptance
 
 If consensus is reached on a specific proposal and we decide to accept it, a
 member of the language team will be chosen to shepherd the implementation.
+
 The implementation will be tracked via two artifacts:
 
-  - A feature specification document
+  - The feature specification document
+  - The feature project
 
-  - A 'implementation' issue.
+### Feature Specification
 
 The 'feature specification' is **a single canonical writeup of the language
-feature** which is to serve as the implementation reference. Feature
-specifications use Markdown format. The file name should be
-`feature-specification.md`, and the feature specification should be located
-inside the `accepted/future-releases/` folder.
+feature** which is to serve as the implementation reference. The feature
+specification should follow the [Proposal Checklist](https://github.com/dart-lang/language/blob/main/doc/life_of_a_language_feature.md#proposal-checklist).
+
+Once we've chosen a specification, we'll update it throughout the implementation
+process. For example, we could add new information to the specification if we
+encounter edge-cases we didn't think of initially or remove parts of it if a
+certain behavior isn't feasible to implement.
 
-The implementation issue (labelled `implementation`) will be filed in the
-language repository for tracking the implementation process. This top of this
-issue must contain links to:
+### Feature Project
 
-  - The related `request` and `feature` issues (if they exist)
+We'll [create a project](https://github.com/orgs/dart-lang/projects) in
```

Comment:
I much prefer using issues over projects; projects get unwieldy fast, and are harder for most to consume. I think especially given GitHub is working on rolling out support for sub-issues, we should not recommend using projects, and just stick with meta-issues for now. That also seems like it's what is being discussed below?

https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/adding-sub-issues

Reply:
I would like to keep this for now. I feel like we've always used projects, even if we do have a meta issue. Plus, as being someone trying to get an understanding of all the open bugs or feedback, it's useful to have them all linked together in one project to look at and they might not always be covered by the umbrella of sub-issues that stem from the one meta issue.

I saw the new sub-issues feature and I haven't had a chance to use it yet. I'll give it a shot for organizing enum shorthands and I'll update the doc once we do.